astropy/time/tests/test_delta.py

```diff
@@ -297,6 +298,45 @@ def test_scales_for_time_minus_time(self, scale1, scale2):
         t2_recover = getattr(t2_recover_t1_scale, scale2)
         assert allclose_jd(t2_recover.jd, t2.jd)
 
+    def test_local_scales_for_time_minus_time(self):
+        """T(X) - T2(Y)  -- does T(X) - T2(Y).X and return dT(X)
+        and T(X) +/- dT(Y)  -- does (in essence) (T(X).Y +/- dT(Y)).X
+
+        I.e., time differences of two times should have the scale of the
+        first time.
+
+        Also tests that subtracting two time scales, one of which is
+        different from local time scale should raise TypeError.
+        """
+        scale1 = 'local'
```

Comment:
Rather than copy the test from above, can we just write 'local'? That makes the test easier to understand, and we're not iterating over different scales here.